pubsublite/spark-connector/spark_streaming_from_pubsublite_example.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+
+def spark_streaming_from_pubsublite(
+    project_number: int, location: str, subscription_id: str
+) -> None:
+    # [START pubsublite_spark_streaming_from_pubsublite]
+    from pyspark.sql import SparkSession
+    from pyspark.sql.types import StringType
+
+    # TODO(developer):
+    # project_number = 11223344556677
+    # location = "us-central1-a"
+    # subscription_id = "your-subscription-id"
+
+    spark = SparkSession.builder.appName("read-app").master("yarn").getOrCreate()
+
+    sdf = (
+        spark.readStream.format("pubsublite")
+        .option(
+            "pubsublite.subscription",
+            f"projects/{project_number}/locations/{location}/subscriptions/{subscription_id}",
+        )
+        .load()
+    )
+
+    sdf = sdf.withColumn("data", sdf.data.cast(StringType()))
+
+    query = (
+        sdf.writeStream.format("console")
+        .outputMode("append")
+        .trigger(processingTime="1 second")
+        .start()
+    )
+
+    query.awaitTermination(120)
```

Comment:
Is this an arbitrary number? An explanation or a comment about this number and/or changing this to a more appropriate number for other cases will help, unless it's self-explanatory for Pub/Sub Lite users.